### DISCUSSION TITLE
Why was I banned

### DISCUSSION THREAD

**Discussion by nailuj29** [vlang/v#9442] [General]
I just got banned from the discord server for seemingly no reason. My tag is nailuj29#4242. Why was I banned?

**Comment by SupremeTechnopriest** [comment]
LOL banned from discord for asking why (@)everyone ping was used in the community channel.  Absolutely pathetic.  Mods need to be fired.

**Reply by nailuj29** [reply]
That's why I was banned?!

**Reply by SupremeTechnopriest** [reply]
Apparently.  Thats why I was banned.  I asked why the (@)everyone and Leo added a 👎  reaction and banned me.

**Reply by SupremeTechnopriest** [reply]
Thought maybe there was a big announcement or something, but no, just bad server configuration letting random people ping 4000 members when ever they feel like it and mods scrambling to cover it up.  Pretty grimy, but whatever I don't care, [zig](https://ziglang.org/) > v anyway.

**Reply by nailuj29** [reply]
That's ridiculous

**Reply by nailuj29** [reply]
Luckily I have an alt, and wasnt at my home IP when I got banned, so I'm gonna try to get everyone that was banned unbanned.

**Reply by SupremeTechnopriest** [reply]
Cool, let me know here if you have any success in that.  This is a pretty bad look.  Maybe the lead devs will see this thread and fire the incompetent mods that are about to destroy their community over some harmless questions.

**Reply by SupremeTechnopriest** [reply]
Right before I was banned they were frantically deleting messages.

**Reply by spytheman** [reply]
I've banned you after you contributed in amplifying the chaos caused by a small mistake by a mod.
Complain to Alex, or not - he may unban you.

**Reply by nailuj29** [reply]
Thats bannable? I didn't even receive a warning. I would've stopped if I was warned.

**Reply by nibble-4bits** [reply]
How can I get unbanned? I just told everyone to chill (without using (@)everyone). That was my only message

**Reply by nailuj29** [reply]
@spytheman you should have warned everyone. If I owned the server, _you_ would be the one receiving a ban

**Reply by spytheman** [reply]
I've written a warning that everyone that continues to write in the next 5 minutes risks banning to stop the chaos.

**Reply by spytheman** [reply]
Alex can ban me at any time. I do not care particularly. It will just free me some time to work on V itself more.

**Reply by nailuj29** [reply]
You should warn individual people before you ban them. You should know this.

**Reply by nailuj29** [reply]
Also, pinging 4 thousand people is not a _small mistake_

**Reply by hysmio** [reply]
So why instantly ban everyone when they clearly didn't see your warning that you didn't put out beforehand? + you're deleting the messages so no one can see others messaging about it

**Reply by spytheman** [reply]
yes it was a small mistake

**Reply by nailuj29** [reply]
Yes, pinging everyone was a small mistake?

**Reply by spytheman** [reply]
Yes, it was a small mistake. Do you have reading comprehension problems?

**Reply by nailuj29** [reply]
No, I fail to see how pinging 4 thousand people for no reason is a small mistake

**Reply by SupremeTechnopriest** [reply]
I didnt see your warning.  The layout was constantly shifting from the deletions, made it hard to read anything. I just asked what was going on and got insta banned lol.

**Reply by SupremeTechnopriest** [reply]
IMO, the steps you should take:

1. Disable the use of (@)everyone server wide except for announcement channels or by admins.
2. Unban everyone that has been banned.
3. Post to this discussion once everyone has been unbanned so we can rejoin the server. 
4. Have a more professional tone when you are dealing with your community. Insinuating someone has a comprehension problem is not very professional and it reflects upon your organization.

**Reply by SupremeTechnopriest** [reply]
If incoming spam is a problem and is preventing you from doing what you need to do in the channel, you can temporarily disable write permissions for the channel until you can get it cleaned up, or enable slow mode.

**Reply by SupremeTechnopriest** [reply]
Maybe put something in the announcement channel that explains what happened and why the channel is temporarily locked.

**Reply by SupremeTechnopriest** [reply]
Also poor @everyone org.  I'll bet they get pinged on all kinds of stuff that don't pertain to them.  Thats why I've been using (@)everyone.  LOL

**Reply by nailuj29** [reply]
There is also many bots that allow you to mute a user so they can't talk in any channel

**Reply by SupremeTechnopriest** [reply]
I understand it can be frustrating, but there are better ways to handle incidents like this.

**Reply by nailuj29** [reply]
@spytheman how can I ping Alex so he sees this?

**Reply by SupremeTechnopriest** [reply]
@medvednikov Hi Alex!

**Reply by nailuj29** [reply]
Thanks

**Reply by nibble-4bits** [reply]
I'd like to get unbanned, my tag is `nibble-4bits#9260`

**Reply by hysmio** [reply]
Would also like to be unbanned. moose#0389

> Yes, it was a small mistake. Do you have reading comprehension problems?

> Alex can ban me at any time. I do not care particularly. It will just free me some time to work on V itself more.

Also, @medvednikov, unfortunately the mods might need to be reevaluated. V is a great project but the way this was handled reflects very poorly on it.

**Comment by hysmio** [comment]
Also banned for asking about using (@)everyone mention?

**Comment by nailuj29** [comment]
In my opinion, this is very immature behavior by the mods

**Comment by JalonSolov** [comment]
The mistake was made by someone who is _not_ a mod, and didn't realize that the (@)everyone tag would do that.  It wasn't even so much a mistake as simply not knowing what effect it would have.

The mods who were dealing with the mess were trying to keep it from getting even further out of control, and in the heat of the moment, banning someone for contributing to the mess is not unexpected, even if their intention was to help... I don't imagine all the messages were being examined carefully for content.

It will be up to Alex to sort out, finally, as he is the only one with the permissions necessary to remove the ban from anyone's id.

**Reply by nailuj29** [reply]
How is banning someone not unexpected?

**Reply by SupremeTechnopriest** [reply]
There are settings to disallow usage of (@)here (@)everyone and (@)role that were not properly set up by mods.  It's not a big deal and is an easy fix. What is a big deal is the banning of people who did not violate any rules without warning and the tone in which moderators were using while talking to the community.  I moderate some large discord servers and I have outlined some ideas of how to better handle these issues in the above thread.

> IMO, the steps you should take:
> 
> 1. Disable the use of (@)everyone server wide except for announcement channels or by admins.
> 2. Unban everyone that has been banned.
> 3. Post to this discussion once everyone has been unbanned so we can rejoin the server.
> 4. Have a more professional tone when you are dealing with your community. Insinuating someone has a comprehension problem is not very professional and it reflects upon your organization.

> If incoming spam is a problem and is preventing you from doing what you need to do in the channel, you can temporarily disable write permissions for the channel until you can get it cleaned up, or enable slow mode.

**Comment by medvednikov** [comment]
Hello,

I was asleep during the incident.

I've disabled @ everyone and @ here for all roles.

Banning users was unnecessary, all bans will be removed.

**Reply by SupremeTechnopriest** [reply]
Let us know when they are removed so we can rejoin. Looking forward to a 1.0 release 🙂

**Comment by medvednikov** [comment]
I've removed all bans, I think I got all of them.

**Reply by SupremeTechnopriest** [reply]
Works for me.

**Reply by nailuj29** [reply]
Works for me too, thank you!

**Comment by medvednikov** [comment]
I've also removed the ability to ban people. Mods can now only mute users.

From now on only I can issue bans.
